### ISSUE TITLE
Improve documentation & add examples

### ISSUE THREAD

**Issue by buzzdan** [clojusc/ltest#16]
i'm having some issues with including ltest and runing it in my project
i find it hard to follow the documentation
it would help to have a sample project or simply explain how to add it to a new lein project

thanks a lot 
Dan

**Comment by oubiwann** [comment]
Yeah, there's no docs whatsoever :-( 

(In fact, I'm surprised anyone's really looking at this project!)

I'll add something to the README ...

**Comment by oubiwann** [comment]
Ah, there was more usage than I had remembered. However, I did add more. Until master or the release/0.2.x branch is updated, the task branch has the latest changes:
 * https://github.com/clojusc/ltest/blob/task/add-more-usage/README.md

Note, however, this does assume knowledge of the Clojure programming language, the `lein` tool, and the `clojure.test` library.

**Comment by buzzdan** [comment]
@oubiwann thanks for the quick reply
can you add a link to the simplest example project you can think off ?
(a git repo example would be perfect)

thanks!
Dan
